packages/api-client/src/components/ApiClient/Request/RequestAuthentication/CardForm.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-  <div class="card-form">
+  <form class="card-form">
```

Comment:
Chrome was upset this wasn't a `<form>`

<img width="911" alt="Google Chrome-2024-05-02-15-41-01@2x" src="https://github.com/scalar/scalar/assets/6374090/551e3570-7e2a-4f53-a7aa-b758ff19766a">